pkgs/by-name/ei/eiffel-studio-bin/package.nix

```
@@ -0,0 +1,66 @@
+{
+  autoPatchelfHook,
+  fetchzip,
+  gcc,
+  gnumake,
+  gtk2-x11,
+  lib,
+  makeWrapper,
+  stdenv,
+  xorg,
+  build ? "103187",
+  libraries ? [ ],
+  platform ? "linux-x86-64",
+  version ? "19.05",
+}:
+
+stdenv.mkDerivation {
+  pname = "eiffelstudio-bin";
```

```suggestion
  pname = "eiffelstudio";
```
We don't name packages bin unless there is a source and bin build and rather set meta.sourceProvenance.

---

i am planning on releasing a source build in the near future, i needed to make a bin build first

---

Then we can replace the bin build with the source build unless there is a very compelling reason that we need both.